### DISCUSSION TITLE
Q: 配置了 TURNSTILE， 出现 报错 Forbidden || Q: TURNSTILE is configured, and an error Forbidden appears

### DISCUSSION THREAD

**Discussion by jerryc127** [walinejs/waline#2029] [Q&A] [answered]
### Describe the bug | 问题描述

配置了 TURNSTILE， 出现 报错 Forbidden 

![image](https://github.com/walinejs/waline/assets/16351105/f3d666f7-5122-43cd-862c-caaf85ccd47f)
![image](https://github.com/walinejs/waline/assets/16351105/c97aec8e-1889-4061-8abd-b68b496b0ba4)

### Website URL | 问题网站

https://butterfly.js.org/messageboard/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

MongoDB

---
### Describe the bug | Problem description

Configured TURNSTILE, there is an error Forbidden

![image](https://github.com/walinejs/waline/assets/16351105/f3d666f7-5122-43cd-862c-caaf85ccd47f)
![image](https://github.com/walinejs/waline/assets/16351105/c97aec8e-1889-4061-8abd-b68b496b0ba4)

### Website URL | Problem Website

https://butterfly.js.org/messageboard/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

MongoDB

**Comment by lizheming** [comment, marked as answer]
前台需要配置 turnstileKey 

---
Front desk needs to configure turnstileKey
